### DISCUSSION TITLE
Datatable View: Row edit feature

### DISCUSSION THREAD

**Discussion by benj35** [primefaces/community#1627] [PrimeVue] [answered]
I want the row editable feature to let only one row get edited at a time. Currently multiple rows can be edited at the same time. Meaning, one could get in the process of editing a row before saving or canceling a previous row edit.

**Comment by slavco86** [comment, marked as answer]
We have had exactly the same issue today and found that row editing functionality is very basic and limited in DataTable.
What we have opted was for is a custom actions column and being in full control of the look and feel of the action buttons... which means also manually handling the editing events, data updates, etc.

The only "native" way of achieving single row editing functionality I found was via a `pt` prop like so:
``` 
<Column
        row-editor
        :pt="{
          rowEditorInitButton: ({ instance: { editingMeta } }) => ({
            disabled: !!editingMeta[1]
          })
        }"
      />
```

reason for `[1]` accessor is because `editingMeta` weirdly enough is not an `Array` as `editing-rows` prop would expect/suggest, but instead is an object with array index keys 🤷‍♂️

**Reply by benj35** [reply]
Thank you sir. This was helpful.

**Comment by jacobg** [comment]
How do you do this PrimeVue 4? I'm trying this:
```vue
    <Column
      :rowEditor="true"
      style="width: 10%; min-width: 8rem"
      bodyStyle="text-align:center"
      :pt="{
        pcRowEditorInit: { disabled: true },
        pcRowEditorSave: ({ instance, props, states, context }) => ({ disabled: true })
      }"
    ></Column>
```
but neither one works
